packages/ecr-credential-provider/ecr-credential-provider.spec

```diff
@@ -0,0 +1,45 @@
+%global goproject github.com/kubernetes
+%global gorepo cloud-provider-aws
+%global goimport %{goproject}/%{gorepo}
+
+%global gover 1.24.1
+%global rpmver %{gover}
+
+%global gitrev b9c8d82db9edc3be45f82f8d5320d006c62aa50c
+%global shortrev %(c=%{gitrev}; echo ${c:0:7})
+
+%global _missing_build_ids_terminate_build 0
+%define debug_package %{nil}
+
+Name: %{_cross_os}aws-ecr-credential-provider
+Version: %{rpmver}
+Release: 1%{?dist}
+Summary: Container image registry credential provider for AWS ECR
+License: Apache-2.0
+URL: https://github.com/kubernetes/cloud-provider-aws
+Source: bundled-v%{gover}.tar.gz
+Source1: v%{gover}.tar.gz
```

Comment:
The actual source of the package should be `Source`

Reply:
Done